x-pack/plugins/reporting/server/usage/types.ts

```diff
@@ -63,7 +63,13 @@ export interface AvailableTotal {
   deprecated?: number;
 }
 
-type BaseJobTypes = 'csv' | 'csv_searchsource' | 'PNG' | 'printable_pdf';
+type BaseJobTypes =
+  | 'csv'
+  | 'csv_searchsource'
+  | 'PNG'
+  | 'PNGV2'
```

Comment:
I guess it should be either `png_v2` or `PNG_v2` for consistency with other types. WDYT?

Reply:
The pre-existing values are `PNG` and `printable_pdf`. So for "consistency" the new export types became `printable_pdf_v2` and `PNGV2`. I guess that was a misreading of the convention on my part.

I agree your suggestion is more consistent. I think we could migrate `PNGV2` to `PNG_v2`, we'd need to update these in our telemetry mappings too. Let's merge this for now and I'll open a separate issue to track that effort (and if we choose to pursue it).